modules/assim.sequential/R/downscale_function_hrly.R

```diff
@@ -20,8 +21,16 @@ SDA_downscale_hrly <- function(nc_file, coords, date, covariates){
   # Timereadable
   time <- nc_data$dim$time$vals
   time_units <- nc_data$dim$time$units
-  time_origin <- as.POSIXct(substr(time_units, 12, 31), format="%Y-%m-%dT%H:%M")
-  time_readable <- time_origin + time * 3600  # Convert hours to seconds
+  time_origin_str <- substr(time_units, 12, 31)
+  time_origin <- ymd_hm(time_origin_str, tz="EST")
```

Comment:
Check timezone. If not specified in a file, standard would be to use UTC/GMT not EST